conf/update.go

```diff
@@ -0,0 +1,147 @@
+package conf
+
+import (
+	"bufio"
+	"fmt"
+	"io/fs"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/afero"
+)
+
+// ConfMap is the representation of a config file with sections that are divided by headers (map[header] == section)
+// each section consists of key value pairs.
+type ConfMap map[string]map[string]string
```

Comment:
Seems like this is the process module configuration. Why not name it like that?

Reply:
so I renamed stuff to `ProcessModuleConfig`, but left the `ConfMap` the same because I renamed the package to `processmoduleconfig` and naming it `processmoduleconfig.ProcessModuleConfig` feels very redundant :)